scripts/create_source_tarball.sh

```diff
@@ -32,7 +32,7 @@ REPO_ROOT="$(dirname "$0")"/..
     fi
     # Add dependencies
     mkdir -p "$SOLDIR/deps/downloads/" 2>/dev/null || true
-    wget -O "$SOLDIR/deps/downloads/jsoncpp-1.7.7.tar.gz" https://github.com/open-source-parsers/jsoncpp/archive/1.7.7.tar.gz
+    wget -O "$SOLDIR/deps/downloads/jsoncpp-1.8.4.tar.gz" https://github.com/open-source-parsers/jsoncpp/archive/1.8.4.tar.gz
```

Comment:
This is slightly unrelated to this PR, but I think this script should also check for the downloaded hash.

Reply:
@chfast is it possible to run `cmake` to download deps only? :)

Reply:
It should be possible by `make jsoncpp-project-download`. If not, you have to "export" this ExternalProject step.

CMake will check the hash anyway, so no need to do it twice.